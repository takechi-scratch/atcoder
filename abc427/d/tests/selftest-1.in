5
2 2 1
AB
1 2
2 1
2 2 1
BA
1 2
2 1
3 3 1
BAB
1 2
2 1
3 2
3 3 1
BAA
1 2
2 1
3 2
2 3 1
BA
1 2
2 1
2 2
