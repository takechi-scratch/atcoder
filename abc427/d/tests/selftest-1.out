Alice
Bob
Bob
Bob
Bob
